tools/windows/install-help/cal/FinalizeInstall.cpp

```diff
@@ -64,6 +64,19 @@ bool updateYamlConfig(CustomActionData &customActionData)
     return true;
 }
 
+bool writeInstallInfo(const CustomActionData &customActionData)
+{
+    std::wstring customInstallMethod = L"windows_msi_gui";
```

Comment:
When running the installer with `/q` we won't have a GUI thus I think the install method should be something else (like `windows_msi_cli`) or be absent as before.